src/cluster_legacy.h

```diff
@@ -17,7 +17,7 @@
 #define CLUSTER_CANT_FAILOVER_WAITING_DELAY 2
 #define CLUSTER_CANT_FAILOVER_EXPIRED 3
 #define CLUSTER_CANT_FAILOVER_WAITING_VOTES 4
-#define CLUSTER_CANT_FAILOVER_RELOG_PERIOD (10) /* seconds. */
+#define CLUSTER_CANT_FAILOVER_RELOG_PERIOD 5 /* seconds. */
```

Comment:
This change makes sense to me but now I wonder if we should shrink this further to 1 second, instead? I don't think having a few more duplicated messages at 1HZ is going to be an issue for anyone but there are two good things IMO

1. we don't have to reason about the magic number of 2s and 4s
2. we get to see the system making progress by watching the message.

Using 1s also covers pretty much all cases as I don't see a reason for using a <1s node timeout, test or prod.

Reply:
good comment, agree, 1s also make sense to me, i am changing it